qiime2/metadata/tests/test_metadata.py

```diff
@@ -968,6 +978,7 @@ def test_merging_two(self):
             {'a': [1, 2, 3], 'b': [4, 5, 6],
              'c': [7, 8, 9], 'd': [10, 11, 12]},
             index=pd.Index(['id1', 'id2', 'id3'], name='id')))
+
```

Comment:
Lots of whitespace changes in this section of this file - was that intentional? If it is for improving readability, let's tackle in a separate (non-squashed) commit.